bin/node-cdx-bom

```diff
@@ -61,7 +61,7 @@ main() {
 
   echo
   echo "Validating BOM..."
-  cyclonedx validate --input-format json_v1_3 --input-file "$output_bomfile" --fail-on-errors
+  cyclonedx validate --input-format json_v1_3 --input-file "$output_bomfile"
```

Comment:
@nealajpatel there's really no reason to have this validation step here if we're not going to fail on errors.